locales/ko.json

```
@@ -62,86 +62,86 @@
     },
     "cancelList": {
         "header": {
-            "city": "City",
-            "clinicName": "Clinic Name",
-            "note": "Note",
-            "prefecture": "Prefecture",
-            "report": "Report",
-            "voucherRequired": "Voucher",
-            "ward": "Ward",
-            "wardResidencyRequired": "Ward Residency",
-            "website": "Website"
+            "city": "도시",
+            "clinicName": "진료소 이름",
+            "note": "노트",
+            "prefecture": "현",
+            "report": "보고",
+            "voucherRequired": "바우처",
+            "ward": "구",
+            "wardResidencyRequired": "구 거주지가 필요합니다",
+            "website": "웹사이트"
         },
-        "moreInfo": "Click for more info",
+        "moreInfo": "자세한 정보를 보려면 클릭하세요",
         "report": {
-            "clinicName": "Clinic Name:",
-            "note": "Note about this clinic",
-            "reason": "Why are you reporting this clinic?",
-            "submitReport": "Report",
-            "title": "Report this data"
+            "clinicName": "진료소 이름:",
+            "note": "이 진료소에 대한 참고 사항",
+            "reason": "왜 이,진료소를 신고하십니까? ",
+            "submitReport": "신고",
+            "title": "데이터 신고"
         },
         "search": "검색",
         "voucher": {
-            "notRequired": "Not Required",
-            "required": "Required"
+            "notRequired": "선택",
+            "required": "필수"
         },
         "wardResidency": {
-            "notRequired": "Not Required",
-            "required": "Required"
+            "notRequired": "선택",
+            "required": "필수"
         }
     },
     "delete": {
-        "dialog": "Are you sure you want to delete ID: {0}"
+        "dialog": "아이디를 삭제하시겠습니까: {0}"
     },
     "donate": {
         "supportMe": "Support me"
     },
     "error": {
-        "homePage": "Home page",
-        "notFound": "404 Not Found",
-        "otherError": "An error occurred"
+        "homePage": "홈페이지",
+        "notFound": "오류 404, 페이지를 찾을 수 없습니다",
+        "otherError": "오류가 발생하였습니다""
```

There was an extra quote here which was breaking the JSON

```suggestion
        "otherError": "오류가 발생하였습니다"
```